prelude/bootstrap.js

```diff
@@ -837,9 +865,14 @@ function payloadFileSync(pointer) {
     return this.type === 1;
   };
 
-  Dirent.prototype.isSocket = function isSocket() {
-    return false;
-  };
+  const noop = () => false;
+  Dirent.prototype.isBlockDevice = noop;
+  Dirent.prototype.isCharacterDevice = noop;
+  Dirent.prototype.isFIFO = noop;
+  Dirent.prototype.isSocket = noop;
+
+  Dirent.prototype.isSymbolicLink = (fileOrFolderName) =>
```

Comment:
@erossignon Isn't this overrided by line 877?

Reply:
Right, well spotted.

Reply:
yes, some more refactoring was also possible